src/prefect/client/schemas/responses.py

```diff
@@ -201,6 +201,11 @@ class FlowRunResponse(ObjectBaseModel):
         description="A list of tags on the flow run",
         examples=[["tag-1", "tag-2"]],
     )
+    labels: objects.KeyValueLabelsField = Field(
+        default_factory=dict,
+        description="A dictionary of key-value labels. Values can be strings, numbers, or booleans.",
+        examples=[{"key": "value1", "key2": 42}],
+    )
```

Comment:
Should this `KeyValueLabelsField` come from `prefect.types` instead of `prefect.client.schemas.objects`?